src/lisp/kernel/init.lsp

```diff
@@ -855,6 +856,7 @@ the stage, the +application-name+ and the +bitcode-name+"
                  `(my-do-time #'(lambda () ,form))))
            t)
 (export 'my-time)
+)
```

Comment:
If he really needs it, it'll be in git's history. It's not like it'll be lost forever.